string object_name           # 物体名称 (如: "cup", "bottle", "book")
geometry_msgs/Point position # 3D位置 (x, y, z)
float32 rotation_angle       # Z轴旋转角 (0-360度)
float32 confidence          # 检测置信度 (0-1)
builtin_interfaces/Time stamp